reference/configuration.html.md.erb

```diff
@@ -262,6 +262,44 @@ In the following example, `Example-Header` will be removed from the final respon
     Example-Multi-Value = ["value1", "value2"]
 ```
 
+### `http_service.proxy_proto_options`
+
+Configure the version of the PROXY protocol that your app accepts. Version 1 is the default.
```

Comment:
The confusing part is that for proxy_protocol to be used, the port handler must be `proxy_proto`

```toml
[[services]]
internal_port = 8080
protocol = "tcp"

[[services.ports]]
handlers = ["proxy_proto"]
proxy_proto_options = { version = "v2" } 
```

btw, now that I think about it, there is no way to set `proxy_proto` port handler on `http_service`, only for the general `services` section

Reply:
I think we have to remove the proxy_proto_options from http_service, it only makes sense in the `services` section where you can control the ports and handlers.

Reply:
I removed and added tests so we are sure it works for `services`  too https://github.com/superfly/flyctl/pull/2630